packages/lua-language-server/bee-tests.patch

```diff
@@ -0,0 +1,16 @@
+--- lua-language-server/3rd/bee.lua/make.lua	2021-08-09 14:37:07.253522941 +0530
++++ lua-language-server-patch/3rd/bee.lua/make.lua	2021-08-10 01:00:56.403996582 +0530
+@@ -24,8 +24,8 @@
+     deps = "bootstrap",
+ }
+ 
+-lm:build "test" {
+-    "$bin/bootstrap"..exe, "@test/test.lua",
+-    deps = { "bootstrap", "copy_script", "bee" },
+-    pool = "console"
+-}
```

Comment:
Same here as with other patch, please remove the code instead of commenting it away